app_dart/test/src/datastore/fake_config.dart

```diff
@@ -132,6 +134,9 @@ class FakeConfig implements Config {
   @override
   int get schedulingShardSize => 5;
 
+  @override
+  int get batchSize => batchSizeValue ?? 5;
```

Comment:
Making this const in config.dart would prevent need to add this stub in the test

Reply:
I need to have customized `batchSize` values to validate from test.